service/kvm_config_resources_v2.go

```diff
@@ -0,0 +1,192 @@
+package service
+
+import (
+	"github.com/cenkalti/backoff"
+	"github.com/giantswarm/certs"
+	"github.com/giantswarm/kvm-operator/service/cloudconfigv2"
+	"github.com/giantswarm/kvm-operator/service/resource/configmapv2"
+	"github.com/giantswarm/kvm-operator/service/resource/deploymentv2"
+	"github.com/giantswarm/kvm-operator/service/resource/ingressv2"
+	"github.com/giantswarm/kvm-operator/service/resource/namespacev2"
+	"github.com/giantswarm/kvm-operator/service/resource/pvcv2"
+	"github.com/giantswarm/kvm-operator/service/resource/servicev2"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/framework"
+	"github.com/giantswarm/operatorkit/framework/resource/metricsresource"
+	"github.com/giantswarm/operatorkit/framework/resource/retryresource"
+	"github.com/giantswarm/randomkeys"
+	"k8s.io/client-go/kubernetes"
+)
+
+type kvmConfigResourcesV2Config struct {
+	// Dependencies.
+
+	CertsSearcher      certs.Interface
+	K8sClient          kubernetes.Interface
+	Logger             micrologger.Logger
+	RandomkeysSearcher randomkeys.Interface
+
+	// Settings.
```

Comment:
Nitpick, this comment is useless, can we remove it? Same for the rest of config structs.

Reply:
Talk with @xh3b4sd :)

Reply:
Why talking to me? I already said we can leave out the comments when everybody knows the drill. We can maybe document how to do it in the `fmt` repo and leave out the comments. I also said I am not sure about keeping the default config functions at all because we tend to not set any defaults and the structures are not nested most of the time. We should maybe just decide this and document it in `fmt`.

Reply:
I was kinda also wondering those comments (that are repeating everywhere). Is there anything behind origin of them?

Reply:
They make sure you put things in the "right" place.

Reply:
I don't care about the comments, that's why I don't want to discuss that, I thought you like them.

> I also said I am not sure about keeping the default config functions at all

+1 This is also upstream approach and we would get aligning which is very nice IMO. The defaults can be set in `New*` functions.

Reply:
Sounds good.